currency/templates/currency/delete.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'home/free_layout.html' %}
+{% block content %}
+
+
+<br>
+<div class="row">
+    <div class="col-md-6">
+        <div class="card card-body">
+            <p>Are you sure you want to delete "{{currency}}"?</p>
+            <form action="{% url 'delete_currency' currency.id %}" method="POST" class="form-group">
+                {% csrf_token %}
+                <a class="btn btn-warning" href="{% url 'currency' %}">Cancel</a>
+                <input class="btn btn-danger" type="submit" value="Confirm">
```

Comment:
It's better to be explicit
```suggestion
                <input class="btn btn-danger" type="submit" value="Delete">
```